packages/label/package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "label",
```

This should be `@littlespoon/label`:

```suggestion
  "name": "@littlespoon/label",
```

---

Updated